### PR TITLE
Fix NetAgent II battery voltage

### PR DESCRIPTION
**Pull request by CirnoT**
- For single phase UPSes, variable name for battery voltage was incorrect hence sensor saved in SQL had "" as OID.
- Additionally another issue was fixed where divisor for battery voltage on single-phase UPSes was set to 1, when SNMP card reports 27.0V as 270 (divisor 10).

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/discovery/sensors/voltage/netagent2.inc.php [hunk 1]
```diff
@@ -389,8 +389,8 @@
 // BATTERY Voltage
 # Set divisor and limit ranges 1 phase UPS systems
 if ($in_phaseNum == '1') {
-    $battery_voltage_oid = '.1.3.6.1.4.1.935.1.1.1.2.2.2.0';
-    $battery_voltage1 = snmp_get($device, $battery_voltage_oid, '-Oqv');
+    $battery_voltage_oid1 = '.1.3.6.1.4.1.935.1.1.1.2.2.2.0';
+    $battery_voltage1 = snmp_get($device, $battery_voltage1_oid, '-Oqv');
```
Comment (PipoCanaja) [inline includes/discovery/sensors/voltage/netagent2.inc.php]: Hi,
This does not seems right. The 1 is not at the same position in the var name ... Did you copy paste in the github GUI ? it is easier (and less error-prone) to use git in CLI ...

Comment (PipoCanaja) [reply includes/discovery/sensors/voltage/netagent2.inc.php]: Should be ``` $battery_voltage1_oid ``` in both lines 392 and 393.

Comment (CirnoT) [reply includes/discovery/sensors/voltage/netagent2.inc.php]: Nah, I used CLI but modified the file without really testing it as I've modified it separately. The code below https://github.com/librenms/librenms/blob/1588b75927d2137b185ab3393e83fba8db946459/includes/discovery/sensors/voltage/netagent2.inc.php#L412 references variable names with '1' in name hence I am assuming that '1' was desired by author of this code for some reason.

Comment (CirnoT) [reply includes/discovery/sensors/voltage/netagent2.inc.php]: Fixed it, now both 3-phase and single phase are the same (with single-phase mirroring working 3-phase)

**Comment by PipoCanaja** [comment]
Looks better now :) Could you also add some test data, to allow the tests to catch any breaking issue with this code ?

File: includes/discovery/sensors/voltage/netagent2.inc.php (hunk 1 above)
Comment (PipoCanaja) [reply includes/discovery/sensors/voltage/netagent2.inc.php]: All good.

**Comment by CirnoT** [comment]
The test data is already there with required OIDs and it seems that test suite did not catch it. I've ran update on sensors module to see what difference would it make but there was none, hence I don't think I need to create new snmprec file?

**Comment by PipoCanaja** [comment]
It means that the current data is does not include the new OIDs you added. This is normal and expected, only the necessary test data is collected, if you now need more than what's currently there, you need to collect it again. 
So the best would be to collect a new variant, and run save-test-data on it, and attach the snmprec and json files to the PR.
Thanx

**Comment by CirnoT** [comment]
Added test data

**Pull request merged**